where in the world/where in the world/MapViewController.swift

```diff
@@ -0,0 +1,154 @@
+//
+//  ViewController.swift
+//  where in the world
+//
+//  Created by Li Zhang on 2020-02-11.
+//  Copyright © 2020 Li Zhang. All rights reserved.
+//
+
+import UIKit
+import MapKit
+
+protocol FavToMapDelegate: class{
```

Comment:
Since this class is titled `MapViewController` and it houses the class with the same name, we expect to see the class definition for `MapViewController` first. 

I would move `protocol FavToMapDelegate: class {` to the bottom of this file, or its own file even. This is nice to do, since potentially that protocol can grow and reduce bloat in this file.